testfile2.py

```
@@ -0,0 +1,818 @@
+    """
+    Renders certificate
+    """
+    if custom_template:
+        template = Template(
+            custom_template.template,
+            output_encoding='utf-8',
+            input_encoding='utf-8',
+            default_filters=['decode.utf8'],
+            encoding_errors='replace',
+        )
+        context = RequestContext(request, context)
```

Ruff (0.11.9)

Undefined name `context`

(F821)